src/Models/Checksum.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace TruckersMP\Models;
+
+class Checksum
+{
+    /**
+     * The checksum DLL.
+     *
+     * @var string
+     */
+    protected $dll;
+
+    /**
+     * The checksum ADB.
+     *
+     * @var string
+     */
+    protected $adb;
+
+    /**
+     * Create a new Checksum instance.
+     *
+     * @param array $checksum
+     */
+    public function __construct(array $checksum)
```

Comment:
Shouldn't the construct rather accept parameters of social sites instead of an array? Like the model `Game`.